jscomp/bsb/bsb_config_parse.ml

```diff
@@ -146,6 +145,46 @@ let extract_reason_react_jsx (map : json_map) =
   |> ignore;
   !default
 
+let extract_react_jsx (map : json_map) =
+  let default : Bsb_config_types.react_jsx option ref = ref None in
+  map
+  |? ( Bsb_build_schemas.react,
+       `Obj
+         (fun m ->
+           match m.?(Bsb_build_schemas.jsx_version) with
+           | Some (Flo { loc; flo }) -> (
+               match flo with
+               | "3" -> default := Some Jsx_v3
+               | "4" -> default := Some Jsx_v4
+               | _ -> Bsb_exception.errorf ~loc "Unsupported react-jsx %s" flo)
+           | Some x ->
+               Bsb_exception.config_error x
+                 "Unexpected input (expect a version number) for jsx"
+           | None -> ()) )
+  |> ignore;
+  !default
+
+let extract_react_runtime (map : json_map) =
```

Comment:
This one I'm not sure about naming and other comments, as I've already forgotten what "classic" and "automatic" means.

Reply:
In fact, probably the name should be `extract_jsx_runtime`.

Reply:
Renamed to `extract_jsx_mode` https://github.com/rescript-lang/rescript-compiler/pull/5484/commits/81029b30f20e8b12b9dff1229228e31daa5e7c5b